sdk/servicebus/service-bus/src/util/utils.ts

```diff
@@ -426,6 +426,15 @@ function buildRawAuthorizationRule(authorizationRule: AuthorizationRule): any {
 }
 
 /**
+ * @ignore
+ * Helper utility to check if given string is an absolute URI
+ * @param url
+ */
+export function isAbsoluteUri(url: string) {
+  return url.startsWith("sb://") || url.startsWith("http://") || url.startsWith("https://");
```

Comment:
Agree, updated.
Out of curiosity I've tested (and included a case in suite) for when the URL inputs to service are in upper case, and it always stores the endpoint prefix in lowercase. 
But, seems to be case sensitive on the entity name and stores it same as in input. (Which would be a bug since the URL and entity name should be treated as case insensitive and preferably all lower case (like it does with endpoint prefix).

Hoping that as part of https://github.com/Azure/azure-sdk-for-js/issues/6670 we'll be able to cover the all possible protocols support in more comprehensive ways when we get to it.

Also, the other comment https://github.com/Azure/azure-sdk-for-js/pull/6382#discussion_r360989426 seems repeat of https://github.com/Azure/azure-sdk-for-js/pull/6382#discussion_r360986035 - should be now addressed in latest changes.